docs/concepts/whatisvpp/performance.rst

```diff
@@ -0,0 +1,35 @@
+.. _performance:
+
+.. toctree::
+
+Performance
+***********
+
+Performance Expectations
+-------------------------
+
+One of the benefits of this implementation of FD.io VPP is its high
+performance on relatively low-power computing. This high level of
+performance is based on the following highlights:
+
+* High-performance user-space network stack for commodity hardware
+* The same code for host, VMs, Linux containers
+* Integrated vhost-user virtio backend for high speed VM-to-VM connectivity
+* L2 and L3 functionality, multiple encapsulations
+* Leverages best-of-breed open source driver technology: DPDK
+* Extensible by use of plugins
+* Control-plane / orchestration-plane via standards-based APIs
+
+Performance Metrics
+-------------------
+
+The FD.io VPP platform has been shown to provide the following approximate
+performance metrics:
+
+* Multiple MPPS from a single x86_64 core
+* >100Gbps full-duplex on a single physical host
+* Example of multi-core scaling benchmarks (on UCS-C240 M3, 3.5 gHz, all memory channels forwarded, simple ipv4 forwarding):
```

Comment:
Remove "* Example of multi-core scaling benchmarks (on UCS-C240 M3, 3.5 gHz, all memory channels forwarded, simple ipv4 forwarding):"
Replace with

* Continuous performance regression testing in FD.io, demonstrates FD.io ongoing commitment to achieving ever performance. 

IPv4 Routed Forwarding (1 thread, 1 core, 10GE).

Incorporate this live graph.
https://docs.fd.io/csit/master/trending/_static/vpp/cpta-ip4-1t1c-x520.html